extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.smallrye.openapi.deployment;
 
+import static io.quarkus.security.spi.SecurityTransformerBuildItem.createSecurityTransformer;
+
```

Comment:
Is this static import needed? It looks like the method is called with `SecurityTransformerBuildItem` on 1262.